src/components/components/Component.js

```diff
@@ -28,7 +28,10 @@ export default class Component extends React.Component {
     this.state = {
       entity: this.props.entity,
       name: this.props.name,
-      nameList: []
+      nameList: [],
```

Comment:
Do we still need nameList or do the two subsequent lists (objectList and backgroundList) cover everything?

Reply:
Still needed due to the value={this.state.nameList.filter(option => option.value == componentData.data)} line in the Select component returned for gltf-model